.github/workflows/publish.yml

```diff
@@ -17,6 +22,18 @@ jobs:
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: 6.0.200
+          
+      - name: Get version
+        id: get-version
+        run: echo "version=$(cat version)" >> $GITHUB_OUTPUT
+        
+      - name: Get branch
+        id: get-branch
+        run: echo "branch=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_OUTPUT
+        
+      - name: Confirm release
+        id: confirm-release
+        run: echo "test=$(git tag --list 'v${{ steps.get-version.outputs.version }}' | wc -l | sed s/\ //g)" >> $GITHUB_OUTPUT
```

Comment:
how does this work exactly? do we need to have both a version and a tag, or?

Reply:
It's kind of the inverse of what we've currently got. It checks if the tag exists, and if it doesn't then it releases. It's a way to avoid releasing the same build twice.

Reply:
Yeah, and documentation I guess, it's nice to be able to see which release was created on which tag, same as it is now.

Reply:
ah. right I didn't notice the last step where we push tag

Reply:
I'm realizing some documentation needs updating, will do that before this is merged.

Reply:
or not, I guess, doesn't look like the release process is documented at all.